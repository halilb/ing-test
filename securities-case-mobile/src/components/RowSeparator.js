import React from 'react';
import { View } from 'react-native';

import { commonStyles } from '../helpers/theme';

const RowSeparator = () => {
  return <View style={commonStyles.rowSeparator} />;
};

export default RowSeparator;
