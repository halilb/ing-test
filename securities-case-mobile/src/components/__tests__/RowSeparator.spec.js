import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import RowSeparator from '../RowSeparator';

it('renders a row separator correctly', () => {
  const snapshot = renderer.create(<RowSeparator />).toJSON();
  expect(snapshot).toMatchSnapshot();
});
