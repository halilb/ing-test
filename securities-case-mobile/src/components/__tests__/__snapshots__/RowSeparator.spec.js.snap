// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a row separator correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgb(200, 199, 204)",
      "height": 0.5,
      "marginLeft": 8,
    }
  }
/>
`;
